NLP/BI-LSTM_CRF_Tensorflow/train.py

```diff
@@ -0,0 +1,111 @@
+from utils import modifying
+import tensorflow as tf
+from model import NerModel
+import pandas as pd
+import tensorflow_addons as tf_ad
+import os
+import numpy as np
+import datetime
+import argsparse
+
+
+######################################################################################################
+
+parser = argparse.ArgumentParser(description="train")
+parser.add_argument("--output_dir", type=str, default="checkpoints/",help="output_dir")
+parser.add_argument("--max_len",type=int,default=50,help="max_len")
+parser.add_argument("--batch_size", type=int, default=64,help="batch_size")
+parser.add_argument("--hidden_num", type=int, default=512,help="hidden_num")
+parser.add_argument("--embedding_size", type=int, default=300,help="embedding_size")
+parser.add_argument("--embedding_file", type=str, default=None,help="embedding_file")
+parser.add_argument("--epoch", type=int, default=50,help="epoch")
+parser.add_argument("--lr", type=float, default=1e-3,help="lr")
+parser.add_argument("--require_improvement", type=int, default=100,help="require_improvement")
+args = parser.parse_args()
+
+########################################################################################################
+# directory for tensorboard
+
+log_dir = "logs/fit/" + datetime.datetime.now().strftime("%Y%m%d-%H%M%S")
+current_time = datetime.datetime.now().strftime("%Y%m%d-%H%M%S")
+train_log_dir = 'logs/gradient_tape/' + current_time + '/train'
+train_summary_writer = tf.summary.create_file_writer(train_log_dir)
+
+#########################################################################################################
+#Preprocessing Data
+data_train = pd.read_csv("./data/Train_ner.csv",engine="python")
```

Comment:
As you deleted `Train_ner.csv` you have to write a code which downloads data from where you have downloaded. You can see other PRs in which they have made ` dataloader.py `. This is just because it is redundant to push your used data. These things will make you repository easy to use for others in their projects. I suppose you do not know how to commit and push data from your local code which you should learn because it will help you maintain your project and debug these kind of bugs.